ew/cmd/item/itemcmds.py

```diff
@@ -553,7 +553,7 @@ async def inventory_print(cmd):
 
 
     # Send the response(s) then remove the target channel from the list of places already receiving inventories
-    await resp_ctn.post()
+    await resp_ctn.postAsFile(filename = cmd.tokens[0])
```

Comment:
do we want to use a file unconditionally? 99% of the time with huge inventories this makes sense, but i still think a character limit could be used so smaller inventories still display normally

Reply:
despite as funny as i think it is to post a file with just the word "Nothing.", i agree